datafusion/tests/sql.rs

```diff
@@ -687,6 +687,30 @@ async fn select_all() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn create_table_as() -> Result<()> {
+    let mut ctx = ExecutionContext::new();
+    register_aggregate_simple_csv(&mut ctx).await?;
+
+    let sql = "CREATE TABLE my_table AS SELECT * FROM aggregate_simple";
```

Comment:
So cool ❤️

Reply:
one thing you can try is to create table as select from values lists